ktor-server/ktor-server-core/jvm/src/io/ktor/server/config/HoconApplicationConfig.kt

```diff
@@ -49,3 +49,10 @@ public fun Config.tryGetString(path: String): String? = if (hasPath(path)) getSt
  * Get list property value for [path] or `null` if missing
```

Comment:
Could you please clarify what this means? Cannot understand `list property value`